synapse_auto_accept_invite/__init__.py

```diff
@@ -28,6 +32,22 @@ def __init__(self, config: InviteAutoAccepterConfig, api: ModuleApi):
         self._api = api
         self._config = config
 
+        should_run_on_this_worker = (
+            config.worker_to_run_on is None and self._api.worker_name is None
+        ) or (config.worker_to_run_on == self._api.worker_name)
+
+        if not should_run_on_this_worker:
+            logger.info(
```

Comment:
This (and the logging below it) feel a bit debuggey to me.

Reply:
I was kind of mixed on that. It doesn't feel verbose enough to be debug; it's a one-off message on startup and it tends to be the kind of thing that will really help someone out if it's not working.
On the other hand, debug output is so verbose that I don't think many people run with it enabled.
However if you really think so anyway then I can change it